pages/download.html

```diff
@@ -91,35 +83,39 @@ <h1>{% blocktrans %}Download Now ({{ unstable_short_version }} Beta){% endblockt
     </p>
 
     <p>
-      {% blocktrans %}Easy-to-use installer packages for Windows, macOS, and Ubuntu are coming soon. Stay tuned!{% endblocktrans %}
+      <b>{% trans "Important:" %}</b> {% blocktrans with stable_link="#stable" %}While Mixxx 2.1 beta has been stable in testing it is not yet recommended for live use unless you are willing to risk a crash. Looking for the stable release? <a href="{{ stable_link }}">Scroll on down.{% endblocktrans %}</a>
     </p>
+  </div>
 
-    {% comment "FIXME when build server is working for 2.1 branch" %}
-
+  <div style="clear: both;"></div>
+  <div class="halfbox_left darkborder winbox">
     <h2><img src="{% static '/static/images/download_windows.png' %}" class="feature_icon">{{ windows }}</h2>
 
-    <p><a href="{{ unstable_win32 }}"
-          onclick="javascript:trackDownload('{{ unstable_win32_analytics_conversion }}');">
-        {% blocktrans %}{{ unstable_long_version }} for 32-bit Windows{% endblocktrans %}</a></p>
+    <p><a href="{{ unstable_builds }}"
+          onclick="javascript:trackDownload('{{ unstable_win_analytics_conversion }}');">
+        {% blocktrans %}{{ unstable_long_version }} for Windows{% endblocktrans %}</a></p>
 
-    <p><a href="{{ unstable_win64 }}"
-          onclick="javascript:trackDownload('{{ unstable_win64_analytics_conversion }}');">
-        {% blocktrans %}{{ unstable_long_version }} for 64-bit Windows{% endblocktrans %}</a></p>
-    <p><small>{% trans "Supported on Windows XP and newer." %} {% blocktrans with platform_update_link="https://support.microsoft.com/en-us/kb/2117917" %}AAC playback requires at least Windows 7 or Windows Vista with a <a href="{{ platform_update_link }}">platform update</a>.{% endblocktrans %}</small></p>
+    <p>Use the newest file ending in "-x64.exe" for 64 bit Windows and the newest file ending in "-x32.exe" for 32 bit Windows. New versions will be made available every time the code is changed, so check back periodically for updates until Mixxx 2.1 is released.</p>
+
+    <p><small>{% blocktrans with platform_update_link="https://support.microsoft.com/en-us/kb/2117917" %}Supported on Windows XP and newer. AAC playback requires at least Windows 7 or Windows Vista with a <a href="{{ platform_update_link }}">platform update</a>.{% endblocktrans %}</small></p>
+  </div>
 
-    <h2><img src="{% static '/static/images/download_mac.png' %}" class="feature_icon">{{ mac_osx }}</h2>
+  <div class="halfbox_right darkborder macbox">
+    <h2><img src="{% static '/static/images/download_mac.png' %}" class="feature_icon" style="margin-top: -7px;">{{ mac_osx }}</h2>
 
-    <p><a href="{{ unstable_osxintel }}"
+    <p><a href="{{ unstable_builds }}"
           onclick="javascript:trackDownload('{{ unstable_osxintel_analytics_conversion }}');">
         {% blocktrans %}{{ unstable_long_version }} for Mac OS X 10.6+ (Intel){% endblocktrans %}</a></p>
```

Comment:
Minimum required version is macOS 10.8
https://github.com/mixxxdj/mixxx/commit/2313e72944ac6e333ffc3351db68ae9bb45d5992#diff-4c2d3d5e7c14ce76e657efbdb8d11d7e